python/test/t_MetaModelAlgorithm_std.py

```diff
@@ -26,5 +26,9 @@
 sample = ot.Sample(
     [[0], [142.857], [285.714], [428.571], [571.429], [714.286], [857.143], [1000.0]]
 )
-d = ot.MetaModelAlgorithm.BuildDistribution(sample)
-print(d)
+distribution = ot.MetaModelAlgorithm.BuildDistribution(sample)
+print(distribution)
+print("distribution (HTML)")
+print(distribution._repr_html_())
+print("distribution (Markdown)")
+print(distribution._repr_markdown_())
```

Comment:
make up your mind

Reply:
I do not understand: Shouldn't we test them both?

Reply:
it was about the naming

Reply:
nevermind, its okay now